src/store/index.js

```diff
@@ -35,6 +37,7 @@ export default new Vuex.Store({
     useIframeWallet: false,
     sdk: null,
     isBackendLive: true,
+    cookiesConsent: { SoundCloud: false, YouTube: false },
```

Comment:
```suggestion
    cookiesConsent: {},
```
can just assign an empty object and assume that if the scope is falsy (undefined) then we don't have consent? then you don't need to duplicate it in `resetState`